Documentation/subcommands/copy.md

```diff
@@ -9,36 +9,37 @@ file/directory to a specified path.
 
 ## Default Mode
 
-When the `-T` flag isn't specified, `acbuild copy` takes any number of arguments
-greater than or equal to 2. The last specified argument is the directory inside
-the ACI to put the files in, and all other arguments are paths on the host to
-copy. If the target directory doesn't exist in the ACI, it will be implicitly
+By default, `acbuild copy` will copy one thing from the host to a specified
+path. It takes exactly two arguments, the first of which is the path on the
+local system to copy from, and the second is the path inside the ACI to copy
+to. If the directory the target path is in doesn't exist, it will be implicitly
```

Comment:
rather "If the target path's parent directory does not exist in the filesystem, it will be implicitly created (along with any necessary parent directories)" or something